project/Dependencies.scala

```diff
@@ -129,10 +129,9 @@ object Dependencies {
       "com.couchbase.client" % "java-client" % CouchbaseVersion, // ApacheV2
       "io.reactivex" % "rxjava-reactive-streams" % "1.2.1", // ApacheV2
       "org.apache.pekko" %% "pekko-discovery" % PekkoVersion % Provided, // Apache V2
+      "org.apache.pekko" %% "pekko-http" % PekkoHttpVersion % Test, // Apache V2
       "com.fasterxml.jackson.core" % "jackson-databind" % JacksonDatabindVersion % Test, // Apache V2
-      "com.fasterxml.jackson.module" %% "jackson-module-scala" % JacksonDatabindVersion % Test, // Apache V2
-      "com.typesafe.play" %% "play-json" % "2.10.0-RC8" % Test, // Apache V2
```

Comment:
Im guessing these were accidentally left in?

Reply:
sure - I've already removed it